src/applications/claims-status/components/Breadcrumbs.jsx

```diff
@@ -1,29 +1,38 @@
 import React from 'react';
+import { Link } from 'react-router';
+import Breadcrumbs from '@department-of-veterans-affairs/formation/Breadcrumbs';
+
+class ClaimsBreadcrumbs extends React.Component {
+  renderBreadcrumbs = childNodes => {
+    const crumbs = [
+      <a href="/" key="home">
+        Home
+      </a>,
+      <a href="/disability-benefits/" key="disability-benefits">
+        Disability Benefits
+      </a>,
+      <Link to="/" key="claims-home">
```

Comment:
Please advise if it'd be better to change this to `<Link to="your-claims" key="claims-home">`

The `routes.jsx` file seems to redirect there anyway, but I'm all about fault tolerance.

Reply:
Either way is fine, as long as it's not causing a full page reload in the browser when you click on it.

Reply:
I'm not sure I'm understanding what's happening here. Isn't this link a duplicate of the link passed as a child to `ClaimsBreadcrumbs` in `AppealLayout`?

Reply:
@jbalboni I had to change this one to get a unit test to pass. This is a duplicate, but this particular file doesn't seem to be used. It's commented out in the `routes.jsx` logic, and I can't find any other uses of.

This was more a defensive move, to ensure things weren't breaking. I am able to see Appeals data on `localhost` reliably, and haven't noticed any unusual links popping up.

Reply:
Ok, sounds good.

I was trying things out in the GIBCT app and based on that, I think you want to change the link here to be `to="your-claims"` to avoid a page refresh.